project_group_setup.py

```diff
@@ -257,11 +280,27 @@ def main(args):
         add_identifier_to_group(gid, type="osggroup", identifier_name=project_name)
```

Comment:
This isn't part of your changes but I think you can have multiple of the same identifier type so we may have to be careful here.

Reply:
The script now only adds an osggroup identifier if one isn't already present.